lighthouse-core/index.js

```diff
@@ -26,9 +26,9 @@ const Config = require('./config/config');
  */
 
 /**
- * @param {string} url
+ * @param {string=} url
  * @param {LH.Flags=} flags
- * @param {LH.Config.Json|undefined} configJSON
+ * @param {LH.Config.Json=} configJSON
```

Comment:
these (and below in runner) were some drive by fixes I noticed since the test is calling `index.js` directly, and the URL is now optional after #5495